data/virt_autotest/autoyast/host_15.xml.ep

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <reg_server>{{SCC_URL}}</reg_server>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
```

Comment:
For reg_server and reg_code, sle15sp4 host may need specific handling here.

In OSD, 15sp4 host tests use the way in testsuite gi-guest_developing-on-host_sles15sp4-kvm, a recent job link https://openqa.suse.de/tests/10941579. The test also use sle-15-SP5-Online-x86_64 medium, which has below settings. And settings `INSTALL_TO_OTHERS` and `REPO_0_TO_INSTALL` are used to guide host installation to use the 15sp4 images.

sle	15-SP5	Online	x86_64	BETA=1, medium setting is
```
BUGZILLA_URL=http://fastzilla.suse.de/short@BUGID@.xml
BUILD_HA=%BUILD%
BUILD_SDK=%BUILD%
BUILD_SES=%BUILD%
ISO_MAXSIZE=737280000
LTP_GIT_URL=https://github.com/linux-test-project/ltp
LTP_KNOWN_ISSUES=http://qam.suse.de/media/downloads/ltp_known_issues.json
QA_HEAD_REPO=http://dist.suse.de/ibs/QA:/Head/SLE-%VERSION%
SCC_REGCODE=INTERNAL-USE-ONLY-1977-6205
SCC_REGCODE_BCL=35e031a536494293
SCC_REGCODE_GEO=07db9ab0d578f087
SCC_REGCODE_HA=223378a848e109bd
SCC_REGCODE_HPC=bdb13b194f1fece2
SCC_REGCODE_LIVE=f86c0b728fbc04c1
SCC_REGCODE_LTSS_15=ba36407f77ddba37
SCC_REGCODE_RT=5ed98f8c543b55b7
SCC_REGCODE_SES=35eb00e8035bf5b0
SCC_REGCODE_SLES4SAP=2882173e156fbd43
SCC_REGCODE_WE=234a924b6c0dc761
SCC_URL=http://all-%BUILD%.proxy.scc.suse.de
SLENKINS_TESTSUITES_REPO=http://download.suse.de/ibs/Devel:/SLEnkins:/testsuites/SLE_15/
```

If SCC_URL, SCC_REGCODE is used, sle15sp4 would be using sle15sp5 registration. Better to change .

Besides, another info from Julie. Related with 12sp5 autoyast host installation ticket. For sle12sp5 host , in OSD test, the MU update is done actually by the way shown in https://openqa.suse.de/tests/10941166#step/update_package/1
.  So actually we can skip registration in sle12sp5 host autoyast installation. @Julie-CAO  Please correct me if I am wrong.

Reply:
> Besides, another info from Julie. Related with 12sp5 autoyast host installation ticket. For sle12sp5 host , in OSD test, the MU update is done actually by the way shown in https://openqa.suse.de/tests/10941166#step/update_package/1 . So actually we can skip registration in sle12sp5 host autoyast installation. @Julie-CAO Please correct me if I am wrong.

Yes, for tests on sle12sp5 host, "SCC_REGISTER=none" is in use for now. but I don't be against to change to do update via scc in the future.

Reply:
1. It uses SCC proxy for developing product. 15sp5 currently.
2. It uses the default: SCC,  for released product.  15sp4 and other 15spx.  It installs the latest packages during os install so you don't need update them later.  update_package might have nothing to do.
http://10.67.183.130/tests/3577

Reply:
Hi Tony, from job 3577, indeed 15sp4 product is installed. It is validated with wrong testsutie gi-guest_developing-on-host_developing-xen. Testsuite gi-guest_developing-on-host_sles15sp4-kvm/xen are the ones for 15sp4 host. 

> It uses the default: SCC, for released product. 15sp4 and other 15spx.

And for the testsuite settings that need changes to work in the way you describe, I guess it will require overwriting SCC_REGCODE and SCC_URL in testsuite settings? @Julie-CAO @waynechen55   Will it affect other parts of registration/update in tests, eg sle15sp5 guest registration on sle15sp4 host, host upgrade, guest upgrade tests? I guess not based on my info. But I may overlook some details.

Besides, if this won't break any others with this way with their confirmation, I agree with it. Any other testsuite changes needed? We may need you to help summarize it on per-testsuite level and share to @Julie-CAO , then she will be able to help with further update on OSD, to let all work as expected :).

Reply:
SCC_REGCODE is the same but set SCC_URL to empty string.  
 https://openqa.suse.de/tests/10985233#

I posted how I triggered  In the ticket

Reply:
Tony, I guess you mean this comment, https://progress.opensuse.org/issues/112508#note-15? Well, in OSD, the tests are automatically triggered, not like qem jobs which we trigger by ourselves(free to customize settings). So for OSD VT jobs, any change needed in settings, we need to figure out a solution -- updating them in testsuite or medium or wherever, in a suitable way.